package.json

```diff
@@ -15,7 +15,7 @@
     "fuzzing": "docker run -it --rm -v $PWD:/src trailofbits/eth-security-toolbox"
   },
   "devDependencies": {
-    "@chainlink/contracts": "^1.4.0",
+    "@chainlink/contracts": "^1.5.0",
```

Comment:
do we want to pin to 1.5.0 so things dont break if there is an update to 1.6? (remove the ^?)